VotingApplication/VotingApplication.Web/Scripts/GVA-Voting.js

```diff
@@ -1,5 +1,5 @@
 ﻿(function () {
-    angular.module('GVA.Voting', ['ngRoute', 'ngDialog', 'ngStorage']).config(['$routeProvider', function ($routeProvider) {
+    angular.module('GVA.Voting', ['ngRoute', 'ngDialog', 'ngStorage', 'GVA.Common']).config(['$routeProvider', function ($routeProvider) {
```

Comment:
This is rather a long line. Would be a bit easier on the eyes if .module and .controller were each on their own line.